docs/sources/write/front-matter/index.md

```diff
@@ -96,23 +96,24 @@ To determine the relative alias, you must first understand the meaning of the cu
 
 For an alias in the page `/docs/grafana/latest/alerting/manage-notifications/`:
 
-- The current directory element (`.`) refers to the directory containing the current page.
+- The current directory element (`.`) refers to the directory containing the current page, _not_ the directory of the current page.
 
-  In this example, this is the page `/docs/grafana/latest/alerting/manage-notifications/`.
+  In this example, this is the page `/docs/grafana/latest/alerting/`.
 
 - The parent directory element (`..`) refers to the parent directory of the current directory element.
 
-  In this example, this is the page `/docs/grafana/latest/alerting/`.
+  In this example, this is the page `/docs/grafana/latest/`.
 
-The following table demonstrates the redirect URL, **REDIRECT**, created when using the relative alias, **RELATIVE ALIAS**, in the source file for **DESTINATION**:
+The following table demonstrates how to redirect **FROM PAGE**, to **TO PAGE**.
```

Comment:
```suggestion
In the following table, the **FROM PAGE** column lists the page that requires a redirect, for example because the page has been moved or no longer exists. The **TO PAGE** is where we want to redirect them, for example the renamed page or where the content has been moved.
The third column shows the alias that should be added to the front matter of the file in the **TO PAGE** column to create the proper redirect.
```

Reply:
I don't think so.

Reply:
I've attempted to rewrite this bit using a list of the columns and your suggestions for explaining each column in https://github.com/grafana/writers-toolkit/pull/396/commits/462fe219617fcc67146f0b876b4f5cc947dbfc9a